Week3/routs.js

```diff
@@ -0,0 +1,98 @@
+const express = require('express');
+const mysql = require('mysql');
+
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'toDoList',
+});
+
+module.exports = function(app) {
+  app.get('/server', (req, res) => res.send('pong'));
+
+  app.post('/todoList/createOne', (req, res) => {
+    const newList = req.body.list_name;
+    const userId = req.body.user_id;
+    const addNewList = `insert into todo_list (user_id,list_name) values ('${userId}','${newList}')`;
```

Comment:
You are making two errors here:
1) You are not using prepared statements. A malicious entity could send a request where the userId is 'Andrej; drop table users' and delete your entire users table
2) according to the sql above the user_id is auto increment. So it should not be set manually, because that defeats the purpose of auto_increment not to mention nothing guarantees uniqueness in the ids that clients send.